src/argilla_server/services/info.py

```diff
@@ -104,20 +114,29 @@ def api_status(self) -> ApiStatus:
         """Returns the current api status"""
         return ApiStatus(
             version=str(version),
+            argilla=self._argilla_info(),
             elasticsearch=self._elasticsearch_info(),
             huggingface=self._huggingface_info(),
             mem_info=self._api_memory_info(),
         )
 
+    def _argilla_info(self) -> ArgillaInfo:
```

Comment:
instead of using a shared variable, maybe is more clear if the method accepts it as argument (because the side effects)

Reply:
```suggestion
    def _argilla_info(self, huggingface_info) -> ArgillaInfo:
```